cri/v1alpha2/cri_types.go

```diff
@@ -17,6 +17,9 @@ type SandboxMeta struct {
 
 	// Runtime is the runtime of sandbox
 	Runtime string
+
+	// Runtime whether to enable lxcfs for a container
+	LxcfsEnabled bool
```

Comment:
How about v1alpha1?

Reply:
@allencloud Support for CRI functionality extensions is only available for v1alpha2, WDYT?

Reply:
OK, I am OK on it. We need to tell all these details to the users. About the change, about incompatibility, or a plan when to abandon v1apha1.